ui/src/app/views/dataset/component.tsx

```diff
@@ -98,6 +106,10 @@ export const DataSet = withCurrentWorkspace()(class extends React.Component<
       selectedCohortIds: [],
       valueSets: [],
       selectedValues: [],
+      save: false,
```

Comment:
Nit: I'd probably call this saving, rather than save, because it is only true while there is a save in progress.

Reply:
(On further look lower down, `creating` might be another good name.